python/tests/api/types/test_type_util.py

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+from iceberg.api.types import (BooleanType, NestedField, StructType)
+from iceberg.api.types import type_util
+from iceberg.exceptions import ValidationException
+
+
+class TestConversions(unittest.TestCase):
```

Comment:
we are using `pytest` is there any reason to wrap this single test in a unittest class?

Reply:
It was originally `unittest` and converted en-masse to `pytest`, so there are still some straggling things that haven't been updated.  It would be appropriate to update anywhere it's still using the older unit tests.

Reply:
👌  will update the test to use pytest.